src/promptflow/promptflow/_sdk/_service/apis/trace.py

```diff
@@ -16,3 +18,13 @@ def get(self, span_id: str):
         trace = get_client_from_request()._traces.get(span_id=span_id)
         print(trace)
         return "Trace", 200
+
+
+@api.route("/list")
```

Comment:
strong type here for swagger generation

Reply:
accoring to Flask RESTX doc, this is doable via adding a schema model, and I will add an example; but I want to double confirm here the strong type is for trace schema to UX (which should be static most of time), instead of what we store in the memory (or say the entity schema SDK/CLI use)?

Reply:
updated with some fileds:

![image](https://github.com/microsoft/promptflow/assets/38847871/7d14405a-244c-403f-a4b9-4e29b88ca85b)

note that if we expect strong type for nested JSON (e.g., `attributes.span_type`), we need to define every model to implement this - which can spend lots of lines to define; also, we need to finalize the contract - currently, all I can say is this is totally doable, just need some code.

Reply:
I noticed you aready did this? do our implementation allow additional fields in attributes?